documentation-support/src/main/java/tech/picnic/errorprone/documentation/BugPatternTestExtractor.java

```diff
@@ -167,7 +168,10 @@ private static void extractReplacementBugPatternTestCases(
          * is safe, because this code is guarded by an earlier call to `#getClassUnderTest(..)`,
          * which ensures that `tree` is part of a longer method invocation chain.
          */
-        MethodInvocationTree inputTree = (MethodInvocationTree) ASTHelpers.getReceiver(tree);
+        MethodInvocationTree inputTree =
+            (MethodInvocationTree)
+                requireNonNull(
+                    ASTHelpers.getReceiver(tree), "Instance method invocation must have receiver");
```

Comment:
All the places in which I had to add `requireNonNull` are genuinely cases where we can't reasonably expect NullAway to infer the non-nullness invariant. :+1:

Reply:
If you want I can file an issue for this one as well, though since it's not a bug as far as this project is concerned: also happy to leave it.

Reply:
I think I figured this one out; it's probably https://github.com/uber/NullAway/pull/1144.